src/test/java/com/epam/tc/hw3/pages/components/LogComponent.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.tc.hw3.pages.components;
+
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LogComponent {
+
+    @FindBy(css = "ul[class='panel-body-list logs'] > li")
+    private List<WebElement> logs;
+
+    @FindBy(xpath = "//*[contains(text(),'Water: condition changed to true')]")
+    private WebElement waterLog;
+
+    @FindBy(xpath = "//*[contains(text(),'Wind: condition changed to true')]")
+    private WebElement windLog;
```

Comment:
Could be simplified and made on parametrized method

Reply:
done